composer.json

```diff
@@ -15,29 +15,37 @@
         }
     ],
     "require": {
+        "fsi/admin-bundle" : "~2.0|~3.0",
         "php": ">=7.1",
-        "symfony/framework-bundle" : "^2.3|^3.0|^4.0",
-        "fsi/admin-bundle" : "^2.0|^3.0@dev"
+        "symfony/config" : "~2.3,>=2.3.12|~3.0|~4.0",
```

Comment:
The precise versions for branches `2.3` were copied from requirements found for `symfony/framework-bundle`.

Reply:
`^2.3.12|^3.0|^4.0`